src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -1395,134 +1631,158 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
+</maml:para>
 	</maml:description>
 	<command:syntax>
 		<command:syntaxItem>
 			<maml:name>New-ServiceControlUnattendedFile</maml:name>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>Name</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The name of the ServiceControl Instance.  This will be used as the Windows Service Name.  The name must be unique for all Windows services and not contain any spaces or special characters.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>InstallPath</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The directory to use for this ServiceControl Instance.  The binary files are copied here.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>DBPath</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The directory that will contain the embedded database files for this ServiceControl Instance.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>LogPath</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The directory that will contain the logs for this instance of ServiceControl.  This should be unique for each instance of ServiceControl and should not be a subfolder of the InstallPath or DBPath.</maml:para>
 				</maml:description>
 				<command:parameterValue required="true" variableLength="false">String</command:parameterValue>
 			</command:parameter>
 			<command:parameter required="false" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>HostName</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The hostname to use in the URLACL (defaults to localhost).  Anyone who can access the ServiceControl URL has complete access to the message data stored within ServiceControl. This is why the default is to only respond to the localhost. Consider carefully the implications of exposing ServiceControl via a custom or wildcard URL.</maml:para>
```

Comment:
Remove the `This is why` portion.

I think @mikeminutillo had a good suggestion on one of the earlier lines that were similar to this.